src/pages/AddItem/index.js

```diff
@@ -80,20 +82,24 @@ export const AddItemPage = () => {
   };
 
   return (
-    <>
-      <Header className="page-header">Add Item</Header>
-      <ContentContainer>
-        <form onSubmit={onSubmit} className="add-form">
-          <AddForm handleForm={handleForm} formState={formState} />
-          <Button type="submit" disabled={formState.productName === ''}>
-            Add Item
-          </Button>
-        </form>
+    <main className="page-container">
+      <div className="image-container">
+        <img src={Image} alt="Shopping" width="80%" />
+      </div>
+      <div className="card-container">
+        <ContentContainer>
```

Comment:
I was wondering about lines 89-90 😄 . What is the difference between `card-container` and `<ContentContainer />`? 🤔

Reply:
The `card-container` to style and place the white card on the page while the component is styling the contents of that white card. I thought about changing the names but thought it would be too big a refactor for this PR and could impact a lot on this week's merge. We can think about making this improvement next week, in another PR, what do you think?

Reply:
Oh, I got it! 😄

I agree with you👍! You have a reason. It might a good idea for the next week. Thank you so much for thinking about it!